### ISSUE TITLE
Game is not loading

### ISSUE THREAD

**Issue by DarthKajar** [tsunamayo/Starship-EVO#2808]
### Short Description of the issue:
Purchased and installed the game today(08/25/2020) and when I launch, I can make it through the initial loading and game menu screens. However, when I click on start a new character, the game launches the loading screen and it just sits there. I have been waiting for over 30 minutes at one point and I never have the game load. I have closed and restarted the game multiple times. 
...

### Steps to reproduce the issue:

...

### Environment
- Starship EVO version: 20w34a
- Graphics Card: GTX-960

### Additional information:

_(Screenshots, Videos, Error logs, Related issues)_

Please also include the `output_log.txt` file. You can find your log here: `%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\`

**Comment by tsunamayo** [comment]
@DarthKajar Hi, I hope this issue was fixed? Otherwise to investigate I will need to output_log! I have the same GFX car so it should run just fine ;).

**Comment by DarthKajar** [comment]
Only thing I am fighting is extreme choppy control and play. I have went through all of the frame rate/resolution options and I still have major issues with smoothness.

Sent from my iPhone

On Aug 28, 2020, at 5:33 PM, Francois Duret <notifications@github.com> wrote:

﻿

@DarthKajar<https://github.com/DarthKajar> Hi, I hope this issue was fixed? Otherwise to investigate I will need to output_log! I have the same GFX car so it should run just fine ;).
